nixos/modules/security/sudo.nix

```diff
@@ -258,12 +285,12 @@ in
             src = pkgs.writeText "sudoers-in" cfg.configFile;
             preferLocalBuild = true;
           }
-          # Make sure that the sudoers file is syntactically valid.
-          # (currently disabled - NIXOS-66)
-          "${pkgs.buildPackages.sudo}/sbin/visudo -f $src -c && cp $src $out";
+          "${cfg.package}/bin/visudo -f $src -c && cp $src $out";
```

Comment:
Where did `buildPackages` go?

Reply:
This breaks cross-compiling, `cfg.package` is for the host platform; this runs at build time which is why it used `buildPackages`